openapi/paths/storefront/subscriptions@{id}@change-items.yaml

```
@@ -0,0 +1,41 @@
+parameters:
+  - $ref: ../../components/parameters/resourceId.yaml
+post:
+  x-badge: Experimental
+  x-products:
+    - Storefront
+  tags:
+    - Storefront orders
+  summary: Change subscription items
+  operationId: StorefrontPostSubscriptionItemsChange
+  x-sdk-operation-name: changeItems
+  security:
+    - CustomerJWT: []
+  description: |-
+    Creates a quote for change items and quantities.
+    Applies the update to subscription when the quote is accepted.
```

suggestion: to add more detail
```suggestion
    Creates a quote for requested changes to items and quantities.
    This operation applies the requested changes to the subscription when the quote is accepted.
```